source/MRTest/MRPolylineTrimWithPlane.cpp

```diff
@@ -0,0 +1,120 @@
+#include "MRMesh/MRGTest.h"
+#include "MRMesh/MRMeshFwd.h"
+#include "MRMesh/MRPolylineTrimWithPlane.h"
+#include "MRMesh/MRPolyline.h"
+#include "MRMesh/MRPlane3.h"
+#include "MRMesh/MRBitSet.h"
+
+namespace MR
+{
+
+TEST( MRMesh, TrimPolylineWithPlane )
+{
+    Polyline3 polyline;
+    std::vector<Vector3f> points = { {0.f, 0.f, 0.f}, {2.f, 2.f, 2.f}, {0.f, 4.f, 0.f} };
+    Plane3f plane( { 1.f, 0.f, 0.f }, 1.f );
+
+    Polyline3 otherPart;
+    DividePolylineParameters params;
+    params.closeLineAfterCut = true;
+    params.otherPart = &otherPart;
+    trimWithPlane( polyline, plane, params );
```

Comment:
input polyline is empty here, is it intentional?

Reply:
outdated code, removed